spreadsheets/scalerank/landuse.csv

```diff
@@ -1 +1,72 @@
-kind,scalerank
+kind,operator,zoom::int,area::int,scalerank
+national_park;park or protected land;battlefield,!United States Forest Service,3,>=300000000,1
+national_park;park or protected land;battlefield,!United States Forest Service,4,>=300000000,1
+national_park;park or protected land;battlefield,!United States Forest Service,5,>=150000000,1
+national_park;park or protected land;battlefield,!United States Forest Service,6,>=150000000,1
+national_park;park or protected land;battlefield,!United States Forest Service,7,>=100000000,1
+national_park;park or protected land;battlefield,!United States Forest Service,8,>=10000000,1
+national_park;park or protected land;battlefield,!United States Forest Service,9,>=5000000,1
+national_park;park or protected land;battlefield,!United States Forest Service,10,>=1000000,1
+national_park;park or protected land;battlefield,!United States Forest Service,11,>=500000,1
+national_park;park or protected land;battlefield,!United States Forest Service,12,>=500000,1
+national_park;park or protected land;battlefield,!United States Forest Service,13,>=100000,1
+national_park;park or protected land;battlefield,!United States Forest Service,14,>=50000,1
+national_park;park or protected land;battlefield,!United States Forest Service,15,>=2000,1
+national_park;park or protected land;battlefield,!United States Forest Service,>=16,+,1
+national_park;park or protected land,!United States National Park Service,4,>=1000000000,2
```

Comment:
1. `park or protected land` is an outmoded v0.10 reference, it should be removed I think?
2. Make sure `!United States National Park Service` matches Olga's change from last week?